fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -0,0 +1,118 @@
+            path = get_strava_storage_path(
+                bucket=InfrastructureNames.bronze_bucket,
+                athlete_id=athlete_id,
+                strava_model=StravaStreams.ACTIVITY,
+                activity_id=activity_id,
+            )
+            write_json_to_storage(path, json.loads(activity.model_dump_json()))
+
+            streams = client.get_activity_streams(
+                activity_id=activity_id, types=[e.value for e in StravaStreams]
+            )
+            stream_time = streams.get(StravaStreams.TIME.value, Stream()).data
```

Ruff (0.8.2)

Undefined name `Stream`

(F821)